### PR TITLE
fix(cs): revoke pending registration on delete request

### PR DESCRIPTION
**Pull request by ovidijusnortal**
also small SS UI fix

Closes XRDDEV-2395

### CODE REVIEW

**Comment by ovidijusnortal** [comment]
[rerun-ci]

**Comment by ovidijusnortal** [comment]
[rerun-ci]

File: src/central-server/admin-service/int-test/src/intTest/resources/behavior/api/managementRequestsApi.feature [hunk 1]
```diff
@@ -14,6 +14,16 @@ Feature: Management requests API
     Then authentication certificate of 'CS:E2E:member-1:SS-X' is deleted
     And security server 'CS:E2E:member-1:SS-X' has no authentication certificates
 
+  @Modifying
+  Scenario: Add/delete not yet approved Authentication certificate
+    Given new security server 'CS:E2E:member-1:SS-X' authentication certificate registered with origin 'SECURITY_SERVER'
+    And management request is with status 'WAITING'
+    When authentication certificate of 'CS:E2E:member-1:SS-X' is deleted
+    And management request list endpoint queried and verified using params
```
Comment (mikkbachmann) [inline src/central-server/admin-service/int-test/src/intTest/resources/behavior/api/managementRequestsApi.feature:22]: Should this be 'Then'?

**Comment by ovidijusnortal** [comment]
[rerun-ci]

**Review by mikkbachmann — APPROVED** [review]

**Review by mikkbachmann — APPROVED** [review]

**Pull request merged**